numcodecs/tests/test_pickles.py

```diff
@@ -43,7 +43,13 @@ def test_repr():
     check_repr("Pickle(protocol=-1)")
 
 
-@pytest.mark.skipif(sys.byteorder != 'little',
-                    reason='Pickle does not restore byte orders')
+# Details on xfail
+# https://stackoverflow.com/questions/58194852/modulenotfounderror-no-module-named-numpy-core-multiarray-r
+@pytest.mark.skipif(
+    sys.byteorder != "little", reason="Pickle does not restore byte orders"
+)
+@pytest.mark.xfail(
+    sys.platform == "win32", reason="Pickle fails to read w/ Windows carriage return"
```

Comment:
Would suggest linking the issue like so

```suggestion
    sys.platform == "win32", reason=(
        "Pickle fails to read w/ Windows carriage return "
        "( https://github.com/zarr-developers/numcodecs/issues/271 )"
    )
```